### PR TITLE
Add weekly adherence report APIs

### PR DESCRIPTION
**Pull request by alxdarksage**

### CODE REVIEW

File: src/main/java/org/sagebionetworks/bridge/models/schedules2/adherence/eventstream/EventStreamDay.java [hunk 1]
```diff
@@ -83,4 +84,10 @@ public void setTimeWindows(List<EventStreamWindow> timeWindows) {
     public void addTimeWindow(EventStreamWindow timeWindowEntry) {
         this.timeWindows.put(timeWindowEntry.getTimeWindowGuid(), timeWindowEntry);
     }
+    public String getLabel() {
+        return label;
+    }
+    public void setLabel(String label) {
+        this.label = label;
+    }
```
Comment (alxdarksage) [inline src/main/java/org/sagebionetworks/bridge/models/schedules2/adherence/eventstream/EventStreamDay.java:92]: Do we need this label? Double-check.

File: src/main/java/org/sagebionetworks/bridge/models/schedules2/adherence/weekly/NextActivity.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    public String getStudyBurstId() {
+        return studyBurstId;
+    }
+    public Integer getStudyBurstNum() {
+        return studyBurstNum;
+    }
+    public LocalDate getStartDate() {
+        return startDate;
+    }
+}
```
Comment (alxdarksage) [inline src/main/java/org/sagebionetworks/bridge/models/schedules2/adherence/weekly/NextActivity.java:60]: Don't we need the label here too?

**Comment by alxdarksage** [comment]
Labels are also a cue to the UI as to how many rows there need to be in this column-major report. So don't hide them, show them.

File: src/main/java/org/sagebionetworks/bridge/models/schedules2/adherence/eventstream/EventStreamDay.java (hunk 1 above)
Comment (alxdarksage) [reply src/main/java/org/sagebionetworks/bridge/models/schedules2/adherence/eventstream/EventStreamDay.java:92]: Yes, it's obviously important in the JSON output....

File: src/main/java/org/sagebionetworks/bridge/models/schedules2/adherence/weekly/NextActivity.java (hunk 2 above)
Comment (alxdarksage) [reply src/main/java/org/sagebionetworks/bridge/models/schedules2/adherence/weekly/NextActivity.java:60]: Added

**Comment by alxdarksage** [comment]
Also adding all labels for a user Understanding the "rowLabels" property.

**Review by jnelsonjava — APPROVED** [review]
Looks good.

**Pull request merged**